src/main/java/picard/analysis/CollectBaseDistributionByCycle.java

```diff
@@ -38,6 +38,7 @@
 
 import htsjdk.samtools.util.StringUtil;
 import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.help.DocumentedFeature;
 import picard.PicardException;
 import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
```

Comment:
Imports don't appear to be sorted in many of these files. This isn't the worst thing but it would be good to have them in canonical order to avoid unnecessary churn in the future.